include/sir/config.h

```diff
@@ -277,7 +277,7 @@
  * data in any color mode (the largest possible sequence, which is:
  * `\x1b[a;fb;m;rrr;ggg;bbb;fb;m;rrr;ggg;bbbm`) plus a null terminator.
  */
-# define SIR_MAXSTYLE 39
+# define SIR_MAXSTYLE 43
```

Comment:
Where'd you come up with this?

Reply:
GCC did.  I can change it back and let CI fail so you can see the exact message.

Reply:
Here's the error: https://gitlab.com/libsir/libsir/-/jobs/4960927984#L70